lib/services/ios-debug-service.ts

```diff
@@ -72,14 +74,15 @@ class IOSDebugService implements IDebugService {
         }).future<void>()();
     }
 
-    private emulatorDebugBrk(): IFuture<void> {
+    private emulatorDebugBrk(shouldBreak? : boolean): IFuture<void> {
```

Comment:
Again, these in my opinion are more like exceptions not common rule. Look at other methods signature.